modules/ROOT/pages/spike-control-reference.adoc

```diff
@@ -8,45 +8,83 @@ endif::[]
+* Window size: 1 second
+* # requests per window: 2
+* Reprocessing delay: 499 milliseconds
+* # retries: 1
 
-* Window size: 1 second.
-* # requests per window: 2.
-* Reprocessing delay: 499 milliseconds.
-* # retries: 1.
+== Requests Timelines
+
+The following diagram illustrates the lifespan of each request accepted by the algorithm:
 
 image::spike-control.png[Accepted,Rejected, Queued Requests Timeline]
 
-Curly brackets delineate the lifespan of each request accepted by the algorithm as follows:
+. The first two requests are accepted and the available quota is now 0. 
++
+The quota increases to 1 after a second (window size) has passed from request #1.
+. Request #3 arrives. 
++
+Because no quota is available, the request is queued for 499 milliseconds.
+. Request #4 arrives. 
++
+Because no quota is still available, the request is queued for 499 milliseconds.
+. Shortly before request #3 is retried, request #1 ages a second and its quota is released, because it falls out of the window.
+. Quota is now 1, and request #3 is reprocessed successfully. 
++
+Quota returns to 0 until request #2 ages 1 second.
+. Request #4 delay concludes and it must be reprocessed.
+Request #4 is rejected. Requests #2 and #3 have not aged enough because there are no more delays.
+. Request #5 arrives.
++
+Because request #2 has already aged more than a second, the request is accepted.
+
+Based on the example, the request delay regulates the spikes in traffic. Consequently, in high contention scenarios the backend continues to function properly (it serves no more than X requests in the last Y milliseconds). 
+
+Users querying the API might still see their requests being served, though with higher latency. The exact configuration for the policy depends exclusively on your API and its throughput.
+
+== Configuring Policy Parameters
+
+When you apply the policy to your API from the UI, the following parameters are displayed:
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| Element | Description | Required
+| Number of Reqs | The number of requests that are allowed in the number of specified milliseconds. | Yes
+|Time Period | The time period, in milliseconds, within which a request must be processed. | Yes
+| Delay Time in Milliseconds | The amount of time for which each request is retained before retrying (in milliseconds) in case there is no quota remaining. | Yes
+| Delay Attempts | The number of times the request is retried before it is rejected. | Yes
+| Queuing Limit | The number of requests that can be queued at the any given time. Queuing a request requires holding on to a thread and an HTTP connection. The queuing limit mechanism protects the API, as it opens up a valve to ensure the API does not go down in case of an attack. | No
```

Comment:
```suggestion
| Queuing Limit | The number of requests that can be queued | No
```
The removed text belongs in the narrative and not in the table of elements and descriptions.
It is conceptual information, not reference information.

Reply:
Done! Thanks!